snuba/datasets/storages/discover.py

```diff
@@ -82,7 +83,7 @@
                     "sentry:dist": "dist",
                     "sentry:user": "user",
                 },
-                "contexts": {"trace.trace_id": "trace_id"},
+                "contexts": {"trace.trace_id": "trace_id", "trace.span_id": "span_id"},
```

Comment:
Please separate the migration part of the PR from any code that would start using the column. 
There is no way to synchronize migration and application change in production. So first we need to apply the migration and then we need to start using the column, or we need to hold on on the PR (which then can be modified) while we apply the production migration.

Reply:
good spot thank you